components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx

```diff
@@ -177,20 +183,29 @@ export const PrebuildDetailPage: FC = () => {
                     </div>
                 )}
                 {error ? (
-                    <div className="flex flex-col gap-4">
-                        <Alert type="error">
-                            <span>Failed to load prebuild</span>
-                            <pre>{error.message}</pre>
-                        </Alert>
-                        <Button
-                            variant="destructive"
-                            onClick={() => {
-                                refetch();
-                            }}
-                        >
-                            Retry
-                        </Button>
-                    </div>
+                    notFoundError ? (
+                        <div className="flex flex-col gap-4">
+                            <Alert type="error">
+                                <span>Failed to load prebuild</span>
+                                <pre>Prebuild not found</pre>
+                            </Alert>
+                        </div>
+                    ) : (
+                        <div className="flex flex-col gap-4">
+                            <Alert type="error">
+                                <span>Failed to load prebuild</span>
+                                <pre>{error.message}</pre>
```

Comment:
[nit] we can replace `{error.message}` into `{notFoundError ? xxx : error.message}`

Reply:
@mustard-mh we have this whole separate case instead of just tweaking the error message because if it's not found we do not want to offer a Refetch button.